src/ServiceBuilder.php

```diff
@@ -180,8 +181,13 @@ public function naturalLanguage(array $config = [])
      * $pubsub = $cloud->pubsub();
      * ```
      *
-     * @param array $config [optional] Configuration options. See
-     *        {@see Google\Cloud\ServiceBuilder::__construct()} for the available options.
+     * @param array $config [optional] {
+     *     Configuration options. See
+     *     {@see Google\Cloud\ServiceBuilder::__construct()} for the other available options.
```

Comment:
Can you add a "see here for other options" link to `ServiceBuilder::translate()` as well? I'd comment inline there, but can't comment where there isn't a change.